docs/roadmap-2021.md

```diff
@@ -1,14 +1,42 @@
+### Roadmaps
+- [Eventing Working Group](#Eventing)
+- [Eventing Sources Working Group](#Eventing-Sources)
+- [Eventing Kafka Working Group](#Eventing-Kafka)
+
 # Eventing
 
 ## Next 3 Months (Short-term)
 
+### Spec Solidification
+
+Review and update documented specification to reflect the existing
+implementation (v1 API).
+
+**GitHub Issue:** https://github.com/knative/eventing/issues/4595
+
+**Owner:** Evan Anderson
+
 ### Knative Eventing 1.0 Ready
```

Comment:
It seems like one of the big items remaining is the conformance work. Do we intend to include that as part of this roadmap item since it links to https://github.com/knative/eventing/projects/19 ?